data.qmd

```diff
@@ -310,6 +324,18 @@ df |> ggplot(aes(clinic_recruiting, fill = clinic_recruiting)) +
   scale_x_discrete(labels = function(x) stringr::str_wrap(x, width = 15))
 ```
 
+df |> ggplot(aes(clinic_recruiting, fill = clinic_recruiting)) +
```

Comment:
You need a line above that starts the code block and says that the code block contains R code...

````
```{r}
```
````

Reply:
On second viewing I'm not sure why the recruiting clinic has been put in a second time, perhaps a copy-paste with the intention of plotting a different variable?